rpcs3/main.cpp

```diff
@@ -524,6 +528,33 @@ int main(int argc, char** argv)
 		Emu.SetConfigOverride(config_override_path);
 	}
 
+	std::string firmware_path;
+
+	if (parser.isSet(arg_installfw))
+	{
+		firmware_path = parser.value(installfw_option).toStdString();
+		if (!fs::is_file(firmware_path))
+		{
+			report_fatal_error(fmt::format("No firmware file found: %s", firmware_path));
+			return 1;
+		}
+	}
+
+	// Force install firmware first if specified through command-line
+	if (fs::is_file(firmware_path))
```

Comment:
This can all be moved into the scope above

Reply:
Fixed in last commit.